app/models/user.rb

```diff
@@ -1,6 +1,6 @@
 class User < ApplicationRecord
   validates :email, uniqueness: true, presence: true
-  validates_presence_of :password, require: true
+  validates_confirmation_of :password, require: true
```

Comment:
This seems awesome!  Would love to hear more about this command.

Reply:
I am going to check on your previous comment and see about the suspicions for password_confirmation before I can explain this :)